EpsilonAuxBMS/Inc/SetContactor/CheckContactorSet.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include "AuxBmsTasks.h"
+#include "stm32f4xx_hal_adc.h"
```

Comment:
Oh, for unit test reasons, you'll need to also include stm32f4xx_hal.h before this.